Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1951,7 +1951,7 @@ DisabledAdaptationsMetaTagEnabled:
 
 DisallowSyncXHRDuringPageDismissalEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
I considered `embedder`, but ended up with `stable` since it previously had an Experiment Feature toggle, and wasn't sure if people used it. I am not opposed to moving it to `shipping`.